base/applications/regedit/framewnd.c

```diff
@@ -1474,17 +1469,11 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
     return result;
 }
 
-/********************************************************************************
- *
- *  FUNCTION: FrameWndProc(HWND, unsigned, WORD, LONG)
- *
- *  PURPOSE:  Processes messages for the main frame window.
- *
- *  WM_COMMAND  - process the application menu
- *  WM_DESTROY  - post a quit message and return
+/* PURPOSE: Processes messages for the main frame window
  *
+ * WM_COMMAND - process the application menu
+ * WM_DESTROY - post a quit message and return
  */
```

Comment:
Would be nice to make it actual SAL2.

Reply:
for a WindProc? Your suggestion is not wrong, but seriously: who is ever going to mess up with that for a simple WndProc? Feel free to do that in a follow-up-PR. As you know my focus is usually in getting stuff more compact, not to write Thomas Manns Buddenbrooks.